definitions/schedule.py

```diff
@@ -36,65 +36,83 @@
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[9]),
             Dataset("italy-full", data_dir="subnational/italy-full/cases",
                     max_wall_clock_minutes=7*24*60,
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[11]),
             Dataset("brazil-full", data_dir="subnational/brazil-full/cases",
                     max_wall_clock_minutes=7*24*60,
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[13]),
             Dataset("colombia-full", data_dir="subnational/colombia-full/cases",
                     max_wall_clock_minutes=7*24*60,
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[15]),
             Dataset("india-full", data_dir="subnational/india-full/cases",
                     max_wall_clock_minutes=7*24*60,
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[17]),
             Dataset("united-states-full",
                     data_dir="subnational/united-states-full/cases",
                     max_wall_clock_minutes=7*24*60,
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[19]),
             Dataset("deaths-full", data_dir="national/deaths-full",
                     max_wall_clock_minutes=7*24*60,
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[21]),
             Dataset("cases-full", data_dir="national/cases-full",
                     max_wall_clock_minutes=7*24*60,
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[23]),
             Dataset("united-kingdom-local-full",
                     data_dir="subnational/united-kingdom-local-full/cases",
                     max_wall_clock_minutes=7*24*60
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[25]),
             Dataset("united-kingdom-local-deaths-full",
                     data_dir="subnational/united-kingdom-local-full/deaths",
                     max_wall_clock_minutes=7*24*60
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[25]),
             Dataset("united-kingdom-local-admissions-full",
                     data_dir="subnational/united-kingdom-local-full/admissions",
                     max_wall_clock_minutes=7*24*60
-                    timeout=3*24*60*60),
+                    timeout=3*24*60*60,
+                    frequency=[25]),
             Derivative("united-kingdom-collated-full",
                        data_dir="subnational/united-kingdom-full/collated",
```

Comment:
Hi @zmaalick - this needed scheduling on the same day of month as the three dependencies - you look like you have tried to do it for the -local- versions (all on the 25th) rather than the national level ones. The id in dependencies should point to the correct dataset id (1st argument). 
Also the dependencies point to "united-kingdom" etc, not "united-kingdom-full" so add the -full suffix

Reply:
This still needs resolving:
1) pointing to the wrong dependencies ( -full versions needed)
2) needs frequency=[23] adding